cmd/main.go

```diff
@@ -34,13 +36,27 @@ import (
 const resetDbStateEnv = "db.resetState"
 const dbFileEnv = "db.dbFile.path"
 const appname = "headers"
+const preparedDb = "db.preparedDb"
+const preparedDbFilePath = "db.preparedDbFile.path"
 
 func main() {
 	vconfig := vconfig.NewViperConfig(appname).
 		WithDb()
 
 	freshDbIfConfigured()
 
+	// Unzip prepared db file if configured.
+	if viper.GetBool(preparedDb) {
```

Comment:
Please update the readme to show that there is a possibility to run it with an additional flag and it will synchronize

Reply:
Thanks! Added two new variables to readme